azure-quantum/azure/quantum/_client/_client.py

```diff
@@ -53,31 +57,47 @@ class QuantumClient:  # pylint: disable=client-accepts-api-version-keyword
     :type workspace_name: str
     :param credential: Credential needed for the client to connect to Azure. Required.
     :type credential: ~azure.core.credentials.TokenCredential
-    :keyword endpoint: Service URL. Default value is "https://quantum.azure.com".
```

Comment:
This will be a breaking change for someone using using this kwarg. Its typical to have a deprecation warning before actually removing for folks to move over to the new api